### ISSUE TITLE
Loading 3d .obj file and displaying in Android Studio 

### ISSUE THREAD

**Issue by vaibhavchincholkar** [artoolkitx/artoolkitx#89]
Hello, 
I am new to ARtoolKitx, I am able to run the basic cube application on android. I want to know if there is any simple way to load the .obj and display obj file on the tracked marker. I tried ARSimpleNativeCarsProj but it wasn't opening properly opening in android studio and it gave me errors which I could not resolve. If anyone could give me a simple and direct way to load and display 3D .obj file in ARtoolKit would be really helpful.

**Comment by SCornu** [comment]
Have you found the way to do that? 
Because i have the same problem i cant load 3D objects for an Android project.

**Comment by vaibhavchincholkar** [comment]
Not really, I have loaded the object vertices, normals and faces using[ object loader](https://github.com/seanrowens/oObjLoader/blob/master/com/owens/oobjloader/parser/Parse.java.). If you just pass the vertices and indices from faces you will see the object. Though, I haven't figure out how to apply a texture to the object.
For loading .obj you can use https://github.com/seanrowens/oObjLoader/blob/master/com/owens/oobjloader/parser/Parse.java.

**Comment by SCornu** [comment]
Hi, I was wondering if you could share the way you load vertices and face indices to show the object in the application, that will be a really good help
